test/goose/brokers/redis/batch_test.clj

```diff
@@ -40,12 +52,103 @@
           batch-state-key (d/prefix-batch batch-id)
           enqueued-job-set (d/construct-batch-job-set batch-id d/enqueued-job-set)
```

Comment:
suggestion: Since you've added a namespace var consider removing this, it is a redundant declaration.